src/main/java/io/appium/mitmproxy/MitmproxyJava.java

```diff
@@ -41,35 +51,25 @@ public void start() throws IOException, TimeoutException, URISyntaxException {
                 .start()
                 .getFuture();
 
-        waitForPortToBeInUse(8080);
-        System.out.println("mitmproxy started on port 8080");
+        waitForPortToBeInUse(proxyPort);
+        System.out.println("mitmproxy started on port " + proxyPort);
```

Comment:
why not to use log4j/sl4j?

Reply:
I just didn't want to rewrite everything, but I will use lombok with @Slf4j